doc/extension/example_library/front-end/src/ColoredLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useDynamicProperty } from "taipy-gui";
+
+interface ColoredLabelProps {
+  value?: string;
+  defaultValue?: string;
+}
+
+const colorWheel = ["#FF0000", "#A0A000", "#00FF00", "#00A0A0", "#0000FF", "#A000A0"]
+
+const ColoredLabel = (props: ColoredLabelProps) => {
+  const value = useDynamicProperty(props.value, props.defaultValue, "");
+  if (!value) {
+    return <span />
+  }
+  const getColor = (index: number) => ({ color: colorWheel[index % 6] });
+  return value.split('').map((letter, index) =>
+    <span style={getColor(index)} >{letter}</span>
```

Comment:
need a key property to the span
and a react fragment around the array
That would be:
return <>
{value.split('').map((letter, index) =>
    <span key={'key' + index} style={getColor(index)} >{letter}</span>)}
   </>